### PR TITLE
Fixed#2237  Messenger button works

### LINKED ISSUE #2237: Messenger option does not work in share image activity
**Actual Behaviour**
Presently the Messenger icon when clicked asks for "Do you really want to share on Messenger" but does nothing when clicked yes

**Expected Behaviour**
 Messenger should open and image must  be pre-loaded in it and ready to share/

**Steps to reproduce it**
1) Open an image and press share 
2) An activity opens with various icons. Select the MESSENGER icon which is second in the left column.
3) A dialog box appears , press OK.
4) Nothing happens

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixed #2237 

Changes: Added function which sends the image to messenger via intent.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java [hunk 1]
```diff
@@ -1001,6 +997,18 @@ public void onClick(DialogInterface dialogInterface, int i) {
...
+        Uri uri = Uri.fromFile(new File(saveFilePath));
+        Intent share = new Intent(Intent.ACTION_SEND);
+        share.setPackage("com.facebook.orca");
+        share.putExtra(Intent.EXTRA_STREAM, uri);
+        share.setType("image/*");
+        share.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+        startActivity(Intent.createChooser(share, caption));
+        triedUploading = true;
+        sendResult(SUCCESS);
+            }
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: Can you properly intend this pls :)

File: app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java [hunk 2]
```diff
@@ -1001,6 +997,18 @@ public void onClick(DialogInterface dialogInterface, int i) {
         AlertDialogsHelper.setButtonTextColor(new int[]{DialogInterface.BUTTON_POSITIVE, DialogInterface.BUTTON_NEGATIVE}, getAccentColor(), alertDialog);
     }
 
+    private void shareToMessenger()
+    {
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: Please move this brace to the above line

**Comment by codepoet2017390** [comment]
@abishekvashok  updated

**Comment by abishekvashok** [comment]
It looks exceptionally good to me, just waiting for the build to pass. After that, I would give a merge for this :)

**Pull request merged**